src/extension/geosearch/src/main/java/org/geoserver/geosearch/web/GeoSearchLayerConfigPanel.java

```diff
@@ -1,41 +0,0 @@
-/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
```

Comment:
This file seems to be still referred by the application context in this module?

````
<bean id="geosearchLayerGroupConfig" class="org.geoserver.web.publish.LayerGroupConfigurationPanelInfo">
    <property name="id" value="geosearchLayerGroupConfig"/>
    <property name="titleKey" value="publish.layergroup.config.geosearch"/>
    <property name="componentClass" value="org.geoserver.geosearch.web.GeoSearchLayerGroupConfigPanel"/>
  </bean>
````